apps/web/src/components/profile/my-profile-nav.tsx

```diff
@@ -25,16 +27,18 @@ export default function ProfileNav({ screen }: ProfileNavProps) {
   const { setRedeemable } = useRedemption()
   const { t } = useTranslation()
 
-  const getCurrentPage = useCallback(
-    (href: string) => {
-      return href === pathname || `${href}/add` === pathname
-    },
-    [pathname]
-  )
+  // const getCurrentPage = useCallback(
+  //   (href: string) => href === pathname || `${href}/add` === pathname,
+  //   [pathname]
+  // )
```

Comment:
Was this going to be used in the future?
```suggestion
```

Reply:
nope, gone